src/main/java/jetbrains/buildServer/iaa/utils/AssignerArtifactDao.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2000-2018 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jetbrains.buildServer.iaa.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.List;
+import jetbrains.buildServer.iaa.common.Constants;
+import jetbrains.buildServer.iaa.common.Responsibility;
+import jetbrains.buildServer.serverSide.STestRun;
+import jetbrains.buildServer.users.User;
+import jetbrains.buildServer.users.UserModelEx;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class AssignerArtifactDao {
+  private UserModelEx myUserModel;
+
+  public AssignerArtifactDao(@NotNull final UserModelEx userModel) {
+    myUserModel = userModel;
+  }
+
+  public void put(STestRun testRun, Responsibility responsibility) {
+    File resultsFile = this.getAssignerResultFile(testRun);
```

Comment:
In this class everywhere could be used `java.nio.Path` API instead of `File` without need to convert `File.toPath`.